### ISSUE TITLE
Create script to locally run PR

### ISSUE THREAD

**Issue by praseodym** [kiesraad/abacus#415]
Locally check out and run a PR using a single command. Something like `gh pr checkout #nnn; cd backend; cargo run &; cd ../frontend; npm run dev`.

**Comment by praseodym** [comment]
Script by @jorisleker:
```bash
cd ./abacus/frontend
npm run build
cd ../backend
cargo sqlx database reset -y
cargo run -- --frontend-dist ../frontend/dist --reset-database --seed-data
```

**Comment by cikzh** [comment]
Fixed in #429

Run:
```sh
./build-and-run
```
Or:
```sh
./build-and-run some-git-branch
```
